src/SourceBuild/content/eng/tools/BinaryToolKit/BinaryTool.cs

```diff
@@ -40,7 +40,7 @@ public partial class BinaryTool
             targetDirectory,
             mode);
 
-        if (mode == Modes.both || mode == Modes.clean)
+        if (mode == Modes.All || mode == Modes.Clean)
```

Comment:
Perhaps a more natural way to implement this is to make `Modes` a flags enum such that `All` is defined to be the combination of `Validate` and `Clean` (i.e. `All = Validate | Clean`). That would then allow you to write `if (mode.HasFlag(Modes.Clean))`